### PR TITLE
Changed vfs.h

### PR DESCRIPTION
**Pull request by omnit3a**
Changed:
- Changed copyright date of license
- Added owner/group id info

### CODE REVIEW

File: include/kan/vfs.h [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
-/* Copyright (c), 2022, KanOS Contributors */
+/* Copyright (c), 2023, KanOS Contributors */
 #ifndef _INCLUDE_KAN_VFS_H__
 #define _INCLUDE_KAN_VFS_H__
 #include <stddef.h>
 #include <stdint.h>
 
 #define VFS_FILENAME_LENGTH 255
 
+typedef uint32_t vfs_group_id;
```
Comment (untodesu) [inline include/kan/vfs.h]: Why typedef?

Why not just leave it as uint32_t and change it to uid_t/gid_t later (when there is uid_t/gid_t defined)

**Review by untodesu — CHANGES_REQUESTED** [review]

File: include/kan/vfs.h (hunk 1 above)
Comment (omnit3a) [reply include/kan/vfs.h]: I made it typedef for code clarity, so that it can be reused later. I can change it to uid_t and gid_t

Comment (untodesu) [reply include/kan/vfs.h]: Why not define them right away, say in uapi/kan/types.h?

**Review by untodesu — APPROVED** [review]
Good for now, in future I'll add types.h with gid_t and uid_t

**Pull request merged**